cg/meta/workflow/balsamic.py

```diff
@@ -489,6 +496,15 @@ def get_verified_config_case_arguments(
 
         return config_case
 
+    def has_case_only_wgs_samples(self, case_id: str) -> bool:
+        case: Case = self.status_db.get_case_by_internal_id(internal_id=case_id)
+        return all(
+            [
+                sample.prep_category == SeqLibraryPrepCategory.WHOLE_GENOME_SEQUENCING
+                for sample in case.samples
+            ]
+        )
```

Comment:
according to SonarCloud, it is a bad practive to feed `all()` with list comprehensions due to risks of short-circuiting